src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.ts

```diff
@@ -466,7 +466,7 @@ export class WorkspaceSavedObjectsClientWrapper {
           })
         ).saved_objects.map((item) => item.id);
 
-        if (options.workspaces) {
+        if (options.workspaces && options.workspaces.length > 0) {
```

Comment:
Shall we add some UT or IT to cover this line?

Reply:
sure, let met add a test case